### PR TITLE
[Fix] ADF-789: List values are replaced with URIs if validation of 'not empty' property failed in Item Properties

### PR DESCRIPTION
**Pull request by clagosarias**
### [ADF-789](https://oat-sa.atlassian.net/browse/ADF-789)

**Preconditions:** A class with 2 properties of Search Input type and one of the properties has 'not empty' option checked; an item created under this class;

Steps to reproduce:

- Go to items tab

- Click on item from Preconditions

- Assign list value for the property that is allowed to be empty in Item Properties

- Leave the property with 'not empty' restriction without any value

- Click 'Save'

**Actual result:** The value in the property without restriction replaced with it's URI.
**Expected result:** The value keeps it's label and is not replaced with URI.

### CODE REVIEW

**Review by habemuscode — APPROVED** [review]
LTGM but ternary conditionals `condition ? firstCase : secondCase` as I remember are not allowed in our code guidelines

**Comment by clagosarias** [comment]
> LTGM but ternary conditionals `condition ? firstCase : secondCase` as I remember are not allowed in our code guidelines

😮  Could you please link me to those code guidelines? thanks! @habemuscode

File: helpers/form/elements/xhtml/AbstractXhtmlSearchElement.php [hunk 1]
```diff
@@ -98,6 +98,29 @@ private function createClientCode(): string
 require(['jquery'], function ($) {
     $baseVariables
 
+    var getPropertyDataAndParseUri = function(callback, initialSelection, multiple) {
```
Comment (adriloma) [inline helpers/form/elements/xhtml/AbstractXhtmlSearchElement.php:101]: Do we need a function-scope declaration here? Or are we limited to ES5 for some reason?

**Review by adriloma — APPROVED** [review]
Code review only. Just check a minor comment.

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following conventional commits
- [x] Commits messages are meaningful
- [x] Commits are atomic

**Comment by Natallia-Radkevich** [comment]
Verified by QA  on http://test-clagos.playground.kitchen.it.taocloud.org:41291/

**Pull request merged**